src/all-env.ts

```diff
@@ -0,0 +1,14 @@
+import { isBrowser } from './utils/is-browser';
+
+/**
+ * Reads all safe environment variables from the browser or all environment
+ * variables from the server.
+ */
+export function allEnv(): NodeJS.ProcessEnv {
```

Comment:
Does this return type work nice in the browser?

Reply:
Yeah, it's just an object.

```
{
  [key: string]: string | undefined;
}
```